src/Form/LinkBlockFormDataProvider.php

```diff
@@ -77,13 +83,15 @@ public function __construct(
         LinkBlockCacheInterface $cache,
         ModuleRepository $moduleRepository,
         array $languages,
-        $shopId
+        $shopId,
+        Configuration $configuration
```

Comment:
I think this is a BC Break 🤔

Reply:
This is a BC Break : the signature of the constructor is broken.

Reply:
@atomiix @Progi1984 

can I use `ConfigurationCore` or do you prefer different solution? we need to get configuration so we can't just treat adapter as optional if we want to use it, right?

Reply:
It's not a problem the BC Break. It must be declared. Like that, we can move the module version to major.

Reply:
Can you give me example? It's about label and information in pull request?

Reply:
Is it ok now @Progi1984 ?

Reply:
It's ok you correctly fill the template :)

Reply:
Perfect :)